package.json

```diff
@@ -46,9 +46,30 @@
     "lint-staged": "^15.4.3",
     "nodemon": "^3.1.9",
     "prettier": "3.5.2",
-    "typescript": "^5.7.3",
+    "typescript": "^5.8.2",
     "vitest": "^3.0.7"
   },
+  "optionalDependencies": {
```

Comment:
This isn't required for the typescript update but fixes an issue with NPM where optional dependencies aren't installed causing missing dependency errors in CI or across different operating systems